shared/functions.js

```diff
@@ -50,6 +50,22 @@ export function makeResponse (
   return {event, data}
 }
 
+export function makeLog (
+  groupId: string,
+  currentLogPosition: string
+): Log {
+  return {groupId, currentLogPosition}
+}
```

Comment:
This can be deleted since the only place this is being used is to pass arguments to `setCurrentGroup` mutation.